src/BotPlutusInterface/Contract.hs

```diff
@@ -215,9 +246,15 @@ writeBalancedTx contractEnv (Right tx) = do
           , "Signatories (pkh): " <> Text.unwords (map pkhToText requiredSigners)
           ]
 
+    let ext = if signable then "signed" else "raw"
```

Comment:
Is this even correct? Seems to be the case from the tests but I am not sure we will always have one of the two files

Reply:
"raw" must be there in each case, "signed" is only available, if we have all the necessarry signing keys to sign the tx. I think this works.
Finish without sign and submit behaviour is not exactly ideal, and we had a few discussions about this with Sam, but didn't really have time to improve it unfortunately...
[nits] It's not at all important, but I think we could move this part (reading the actual tx and txId from file and changing the filenames) between the build and sign steps, so we don't have to do all these conditionals.